splicemachine/spliceengine
# DB-9502 Run FLUSH as part of MAJOR_COMPACTION

# Short Description
Calls flush before major compaction

# Long Description
in the method SYSCS_PERFORM_MAJOR_COMPACTION_ON_TABLE partition.flush() is called before partition.compact(true)

# How to test
execute CALL SYSCS_UTIL.SYSCS_PERFORM_MAJOR_COMPACTION_ON_TABLE([SCHEMA_NAME]); in sqlshel and check, that compaction is executed and no errors.
